src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -405,16 +433,24 @@ private function createFormatException($message)
         return new FormatException($message, new FormatExceptionContext($this->data, $this->path, $this->lineno, $this->cursor));
     }
 
-    private function doLoad(bool $overrideExistingVars, string $path, array $paths): void
+    /**
+     * @param bool $envMode Silently ignore not existing files in $paths (but not the one in $path)
+     */
+    private function doLoad(bool $overrideExistingVars, string $path, array $extraPaths, bool $ignoreMissingExtraPaths = false): void
     {
-        array_unshift($paths, $path);
+        array_unshift($extraPaths, $path);
```

Comment:
I'd suggest moving this outside of the doLoad method - and have this method accept `$paths` directly.

Reply:
It means duplicating this line 3 times? Does is it worth it?